### ISSUE TITLE
PHP Warning in Provider/OpenAI.php

### ISSUE THREAD

**Issue by notdefine** [inspector-apm/neuron-ai#295]
The Provider/OpenAI.php show a PHP Warning when calling an Agent. The Agent works fine, except this warning.

```
PHP Warning:  Undefined array key "content" in /app/vendor/inspector-apm/neuron-ai/src/Providers/OpenAI/OpenAI.php on line 129                           
```

This is the dump($message) from https://github.com/inspector-apm/neuron-ai/blob/2.x/src/Providers/OpenAI/OpenAI.php#L96

```php
^ array:3 [                                                                                                                                              
  "role" => "assistant"
  "tool_calls" => array:1 [
    0 => array:3 [
      "function" => array:2 [
        "arguments" => "{"order_number":"ABC-532698-02","system_id":"ABC"}"
        "name" => "get_order_details"
      ]
      "id" => "call_YDetcgy5Rmor1NmatoCH794i"
      "type" => "function"
    ]
  ]
  "annotations" => []
]
```

The array has no content attribute.

**Comment by ilvalerione** [comment]
Fixed in 2.2.1
